uint32 	ctrl_method

float32 turn_angle
float32 base_velocity
int32  	steering_motor_pos
float32 left_steer_angle
float32 right_steer_angle
uint16  left_steer_adc_val
uint16  right_steer_adc_val
int16  	drv_motor_pwm_duty


uint32 CtrlMethod_TurnAndle    		= 1
uint32 CtrlMethod_BaseVelocity		= 2
uint32 CtrlMethod_SteerMotorPos    	= 4
uint32 CtrlMethod_LeftSteerAngle   	= 8
uint32 CtrlMethod_RightSteerAngle  	= 16
uint32 CtrlMethod_LeftSteerAdcVal  	= 32
uint32 CtrlMethod_RightSteerAdcVal 	= 64
uint32 CtrlMethod_DrvMotorPwmDuty 	= 128

